src/components/views/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -0,0 +1,21 @@
+import { Add } from '@material-ui/icons';
+import { FormattedMessage } from 'react-intl';
+import { Box, Button } from '@material-ui/core';
+
+export interface ViewDataTableToolbarProps {
+    onColumnCreate: () => void;
+}
+
+const ViewDataTableToolbar: React.FunctionComponent<ViewDataTableToolbarProps> = ({ onColumnCreate }) => {
+    return (
+        <Box display="flex" justifyContent="space-between">
+            <Box />
```

Comment:
Why the empty box? Is it just for the layout? Then maybe `justifyContent="flex-end"` could be an alternative?